mono-api/mono-api.js

```diff
@@ -0,0 +1,74 @@
+'use strict'; 
+
+const http = require('https');
+const BASE_URL = 'api.monobank.ua';
+/*
+    this token should be passed from repository layer, it's here just for tests
+*/ 
+const TOKEN = 'uMLmvCFm6kA4N8mrflxayXMcZPqFUrxpm9q_CxBsMkaY';
+
+function createRequestBody(path, token) {
```

Comment:
But this function creates not only body, but headers also, doesn't it?

Reply:
ok, I'll change it to `createRequest`